tox.ini

```diff
@@ -3,11 +3,12 @@ envlist = py{35,36}-test, checkstyle, docs
 
```

Comment:
I can do this, but what will it change?

Reply:
This line determines which envs are executed, if you dont provide any via cli parameter. (`tox` instead of `tox -e documentation`).

```
Determining the environment list that tox is to operate on happens in this order (if any is found, no further lookups are made):

* command line option -eENVLIST
* environment variable TOXENV
* tox.ini file’s envlist
```
http://tox.readthedocs.io/en/latest/config.html